dns-server-api/src/lib.rs

```diff
@@ -129,17 +129,52 @@ pub trait DnsServerApi {
     #[endpoint(
         method = GET,
         path = "/config",
+        operation_id = "dns_config_get",
+        versions = "1.0.0".."2.0.0"
```

Comment:
These should be using the constants generated by the `api_versions!` macro.  Same below.

Reply:
yup, done in https://github.com/oxidecomputer/omicron/pull/8047/commits/31691bbbeb4f00836e634c41d0599b7b944c0876